src/aind_metadata_service/labtracks/query_builder.py

```diff
@@ -39,14 +39,10 @@ class LabTracksQueries:
     """Class to hold sql query strings for LabTracks"""
 
     @staticmethod
-    def procedures_from_subject_id(subject_id: str) -> str:
+    def procedures_from_subject_id() -> str:
         """
-        Retrieves the information to populate metadata about subjects.
-
-        Parameters
-        ----------
-        subject_id : str
-            This is the id in the LabTracks Task_Set table
+        Retrieves the information to populate metadata about procedures.
```

Comment:
I updated the docstring to say "populate metadata about **procedures**" rather than "**subjects**", is that correct?